chef/site-cookbooks/wca/templates/env.production.erb

```diff
@@ -15,3 +15,5 @@ RECAPTCHA_PUBLIC_KEY=<%= @secrets['RECAPTCHA_PUBLIC_KEY'] %>
 RECAPTCHA_PRIVATE_KEY=<%= @secrets['RECAPTCHA_PRIVATE_KEY'] %>
 NEW_RELIC_LICENSE_KEY=<%= @secrets['NEW_RELIC_LICENSE_KEY'] %>
 GOOGLE_MAPS_API_KEY=<%= "AIzaSyA-kN3GCsG78MnAp16CIRft221XNmioovk" %>
+GITHUB_BOT_ACCESS_TOKEN=<%= "thewca-bot" %>
```

Comment:
nit: I have no idea why `GOOGLE_MAPS_API_KEY` is written the way it is above. I would have expected this to just be the following instead:

```
GITHUB_BOT_ACCESS_TOKEN=thewca-bot
```